src/core.js

```diff
@@ -309,6 +309,13 @@ $.extend( $.validator, {
 				this.element( element.parentNode );
 			}
 		},
+		onchange: function( element, event ) {
```

Comment:
I am not sure this fix will work in all situations. 

this code will change only the "default-onchange" implementation. in case someone has overridden this option he will not benefit from your change.